### ISSUE TITLE
End of folder announcement

### ISSUE THREAD

**Issue by ranetto** [jcsteh/osara#907]
Hi all, when I'm at the end of a folder, or above all at the end of an internal folder, I always get confused, because with the new changes on the announcement, I always jump over, as fast browsing I hear people say, unlike what happens for the start of a folder, first the end track name and then the announcement of the end of folder or internal folder. This complicates my work a lot because I can never understand when a folder ends: I ask you if the start of a folder or the end of a folder, even if internal, could always be announced before the name of the track that acts as start or end folder. I would like this very much and it would speed up my work a lot, and I also stress that the current behavior is inconsistent as it is not the same in the folder start announcement, where the folder start is announced and then if the same is open or closed and after the track name, while for the end the track name or the folder status is announced first. So the ideal for me would always be the consistent condition of the folder first and the track name last. Thank you all.

**Comment by ScottChesworth** [comment]
Are you sure your OSARA is up to date dude? The end of folder report always comes before the track name here even when the folders are nested, I can't find a way to get the behaviour you've described.

**Comment by ranetto** [comment]
hello I think so, I think I downloaded it last week:
I'll try to download it again today and let you know.
Thank you

**Comment by Lo-lo78** [comment]
@ranetto 
Yes, I understand what you mean.
You want it to say end folder first and then the folder name and then the track name.
Is this what you mean?

**Comment by ranetto** [comment]
exactly like that, it would be great, otherwise I get confused and I can never figure out where the folders end up.

**Comment by ScottChesworth** [comment]
Ignore my comments above, think I must've been building an out of date branch when the issue was first opened.
I want to experiment here in the hope that we can make these reports clearer. It's easy to reorder the information, but I think there's a bigger issue in that, especially when "Report track numbers" is checked by default, there's a long stream of info without any pauses, and IMO that's what makes it hard to follow. All components of that stream seem likely to be relevant to someone.
@jcsteh, I'd like to experiment with carefully introduced pauses, is there a reasonably universal fairly sweat-free way to implement that? Adding commas in a few places would work, pausing at those seems widely supported, but I wonder if there's a better way, something that wouldn't impact the inexplicable amount of people I encounter with their screen reader punctuation verbosity set super high at all times?

**Comment by jcsteh** [comment]
I mean, if you have punctuation set that high - I always do, so I'm one of these strange people of which you speak - I feel like you accept what comes along with that and you're probably using a pretty high speech rate too. So, my feeling is "deal with it". But to answer your question, there isn't a way to do this from OSARA, no.

**Comment by ScottChesworth** [comment]
Gotcha. Any thoughts on whether there's a better symbol than Comma for this? Ideally something that's quick to say if punctuation is enabled and only generates a short-ish pause across most synths?

**Comment by jcsteh** [comment]
Comma is what I would use. I don't have a better suggestion for you.
